### PR TITLE
Jamf & Chef workflows

### PR DESCRIPTION
**Pull request by amadotejada**
## Discord username (optional) please include so we can attribute you with our Contributor role (like so elvis#4747)

Discord: Amado#4572

## Description of changes (updated or new workflows)
New JAMF and Chef workflows for Client Platform Engineers
Commonly, used during testing new policy/cookbook deployments

- run a Jamf policy manually by trigger name
- run a Jamf policy manually by ID
- run a Jamf recon for push inventory info
- run a Chef cookbook manually

### CODE REVIEW

File: specs/chef/run_cookbook_manually.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Manually run a cookbook by name
+command: sudo chef client -o recipe['{{cookbook}}'] 
+tags:
+  - macos
+  - linux
```
Comment (alokedesai) [inline specs/chef/run_cookbook_manually.yml:6]: Mind removing the `macos` and `linux` tags here? From our experience, the more specific tags to the workflow, the better

File: specs/chef/run_cookbook_manually.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+name: Manually run a cookbook by name
+command: sudo chef client -o recipe['{{cookbook}}']
```
Comment (alokedesai) [inline specs/chef/run_cookbook_manually.yml:3]: Is the `sudo` necessary here?

File: specs/jamf/manually_initiate_policy_custom_id.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+name: Manually Initiate a Policy via policy ID
```
Comment (alokedesai) [inline specs/jamf/manually_initiate_policy_custom_id.yml:2]: could we be consistent with capitalization all of these names?

File: specs/chef/run_cookbook_manually.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+name: Manually run a cookbook by name
+command: sudo chef client -o recipe['{{cookbook}}'] 
+tags:
+  - macos
+  - linux
+  - chef
+description: Use to fun a cookbook manually
```
Comment (alokedesai) [inline specs/chef/run_cookbook_manually.yml:8]: Is this supposed to be `run` instead of `fun?`

**Review by alokedesai — COMMENTED** [review]
Overall looks great! Just a few small comments

File: specs/chef/run_cookbook_manually.yml (hunk 1 above)
Comment (amadotejada) [reply specs/chef/run_cookbook_manually.yml:6]: no problem

File: specs/chef/run_cookbook_manually.yml (hunk 2 above)
Comment (amadotejada) [reply specs/chef/run_cookbook_manually.yml:3]: For that command, yes

File: specs/chef/run_cookbook_manually.yml (hunk 4 above)
Comment (amadotejada) [reply specs/chef/run_cookbook_manually.yml:8]: oops

**Review by alokedesai — APPROVED** [review]
LGTM! Thank you for submitting!

**Pull request merged**